_shows/04_05/the_trial.md

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
In future can we move files with `git mv old_path new_path` as this tells git that the file has moved and keeps the history intact. Here it thinks this is a brand new file.